### ISSUE TITLE
Migrate this repo to the docs repo then close it

### ISSUE THREAD

**Issue by jackparmer** [flojoy-io/apps#6]
@Angchau brought up a good point that this repo of example apps is pretty redundant with the docs repo:
https://flowjoy.slack.com/archives/C03KH05RZLK/p1683665877179159?thread_ts=1683662602.379009&cid=C03KH05RZLK

Every app folder that has a README describing what the app does (like https://github.com/flojoy-io/apps/pull/5/files#diff-b4b30a08c345bcbc7d3a521cb317e70fa5fac2ba01a62cb8bbd16924717b5b9d) will have to  appear under the "Tutorials" section of the docs:

<img width="277" alt="image" src="https://github.com/flojoy-io/apps/assets/1865834/ddba386c-f983-465f-b8f9-74c8fdbc27a4">

This way, when a contributor adds a new app example in the docs repo, they will automatically show up under the tutorials dropdown, without having to repeat ourselves in multiple places

For consistency, the Tutorials dropdown should be reorganized to match the current ontology of the nodes organization in Flojoy: https://github.com/flojoy-io/nodes

So, in the docs sidebar, 
-Metrology will be renamed INSTRUMENTS (and Camera Example should go under there)
- "Signal processing example" will go under [GENERATORS/SIMULATIONS](https://github.com/flojoy-io/nodes/tree/main/GENERATORS/SIMULATIONS)

Any tutorial that does not fall under one of these node categories... ...
<img width="272" alt="image" src="https://github.com/flojoy-io/apps/assets/1865834/517cae82-0873-473c-85d0-ca6548abb600">
... ... can go under a General folder (eg, Flojoy UI introduction and Looping)

I can't remember if apps is a sub-repo of `studio` - the startup script  will have to be updated to remove if so

the app txt files themselves could be put in an `apps` folder at the root of the docs repo, that the md files reference through their [front matter](https://jekyllrb.com/docs/front-matter/)?

Please LMK if I can clarify anything

**Comment by dstrande** [comment]
I was thinking that 'Tutorials' in the docs will be more in depth with images and such

And not even necessarily about an app but general information about CRTLS, UI, etc.

**Comment by Angchau** [comment]
Okay but then what would be in the app folder, only an example app with the node? Because the tutorials are showing examples of apps in order to use the node, I agree with Jack we should move everything to the doc repo and why not add example apps with the specific node at the end of the tutorial or in the same folder?  I'll do that for my nodes and you'll tell me if it's good. I'll also try to modify the Docs sidebar in order to match the Flojoy ontology of the node organization

**Comment by dstrande** [comment]
I think it possible we'll have _hundreds_ of apps so having them all in tutorial section is a bit excessive.

Maybe make a new section? In my mind the tutorial section should just be on boarding for new users and if they see too many tutorials it may be confusing.

**Comment by Angchau** [comment]
Yes, I agree with that too, maybe a category for custom nodes tutorials. The thing is that for an Instrument node such as LabJackU3, explaining how the example app is working (Which is mandatory when you want to PR) is the same as writing a tutorial for the node. I think this documentation could go under the new category "custom nodes tutorials" which would be different from "classical nodes tuto" where you'll find tuto about looping and general stuff. Let's see what @jackparmer thinks is best.

**Comment by Angchau** [comment]
Because now the apps folder is only used to review how the new nodes are integrated into an app right?

**Comment by Angchau** [comment]
Currently, in each apps folder, there are 3 files, an app example.txt, an empty file called Description, and a Readme file with only one line to tell what the node function is. I am not sure if this is very useful for anyone because if you read the tuto for these nodes you'll have all the info to use them in your own app.

**Comment by jackparmer** [comment]
maybe one top section "Tutorials" that is for general  onboarding like Dallas said (the same as the one we have now)Then another section after that called "app demos", which has the same subcategories that nodes do. so you could navigate into the instrumentation sub folder, for example, and look at all of the app demos related to stepper motors (for example) and see if there is one close to what you want to builddoes that make sense?Envoyé de mon iPhoneLe 10 mai 2023 à 2:15 PM, Angchau ***@***.***> a écrit :﻿
Currently, in each apps folder, there are 3 files, an app example.txt, an empty file called Description, and a Readme file with only one line to tell what the node function is. I am not sure if this is very useful for anyone because if you read the tuto for these nodes you'll have all the info to use them in your own app.

**Comment by Angchau** [comment]
Yes, this makes a lot of sense, I think it would be better like that. So maybe the third PR when contributing for a custom node will need to be in the repo Flojoy/docs instead of Flojoy/studio/apps (We would be able to delete completely this repo after switching everything to the docs section). Now I am editing my first tuto (Labjack, Phidget, IV curve) because all my images with the old CTRL panel are outdated.

**Comment by jackparmer** [comment]
yep, exactly you'll need to update the PR checklist template Envoyé de mon iPhoneLe 10 mai 2023 à 3:36 PM, Angchau ***@***.***> a écrit :﻿
Yes, this makes a lot of sense, I think it would be better like that. So maybe the third PR when contributing for a custom node will need to be in the repo Flojoy/docs instead of Flojoy/studio/apps (We would be able to delete completely this repo after switching everything to the docs section). Now I am editing my first tuto (Labjack, Phidget, IV curve) because all my images with the old CTRL panel are outdated.

**Comment by Angchau** [comment]
Hey Guys, I updated the sidebar as discussed here, I created some specific categories and moved the existing tutorials into these categories, I was not sure about how to place the signal processing tuto because it uses a lot of nodes. Also, I saw that we have a few custom nodes such as: "Object detection", "Loaders", "Timers", etc.. without any tuto to explain how they work. Is it something we need to address ? I think that we can make some app example that would use many nodes and explain how each of them works in the example (It's already the case for Signal processing with LIN, SIN and Add node).

**Comment by jackparmer** [comment]
@Angchau can you please organize the sidebar demo apps according to this?
https://docs.google.com/spreadsheets/d/1vl37XMAZpx0XkkgpZ78J5VGSBRlga8Iko9_iRPEHo5w/edit#gid=1734985092

**Comment by jackparmer** [comment]
Apps can now be migrated to the `docs` repo by using this template:
https://docs.flojoy.io/nodes/transformers/arithmetic/ADD/
https://github.com/flojoy-io/docs/tree/main/docs/nodes/transformers/arithmetic/ADD
